.github/workflows/tests.yml

```diff
@@ -102,7 +102,13 @@ jobs:
 
     strategy:
       matrix:
-        image: ['latest-develop', 'latest-rc']
+        image: ['latest-rc']
+        required: ['required']
```

Comment:
I believe now that we have `continue-on-error: ${{ matrix.required == 'optional' }}`, we wouldn't even need this line, but let's not over-engineer

Reply:
plus it makes the output nicer in the dashboard!